app/models/grade.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+class Grade < ApplicationRecord
+  has_many :classroom_grades, dependent: :destroy
```

Comment:
I'm not sure we want to enable destroying 🤔 

Like teacher deletes 5th grade, it's a big issue, because we lose data.

```
has_many :classroom_grades, dependent: :restrict_with_error
```